runtime/compilers/rillv1/connectors_test.go

```diff
@@ -68,6 +73,12 @@ notify:
     - channel-test
     users:
     - user@example.com
+`,
+		// Dedicated S3 connector
+		"/connectors/s3-dedicated.yaml": `
+driver: s3
+name: s3-dedicated
```

Comment:
The name defaults to the filename without extensions (for all resources), so not necessary to set explicitly